sprint-mission-6.sql

```diff
@@ -0,0 +1,198 @@
+-- # 초급 문제
+
+-- 1. `orders` 테이블에서 모든 주문을 조회하세요.
+SELECT * FROM orders
+-- 2. `orders`테이블에서 `id` 가 `423`인 주문을 조회하세요.
+select * from orders where id = 423;
+
+-- 3. `orders` 테이블에서 총 주문 건수를 `total_orders`라는 이름으로 구하세요.
+select COUNT(*) as total_order from orders
+
+-- 4. `orders` 테이블에서 최신 순으로 주문을 조회하세요. (`date`, `time` 컬럼이 분리되어 있다는 점에 주의)
+select * from orders 
+order by date desc, time desc;
+
+-- 5. `orders` 테이블에서 오프셋 기반 페이지네이션된 목록을 조회합니다. 페이지 크기가 10이고 최신순일 때, 첫 번째 페이지를 조회하세요.
+select * from orders 
+offset 0 limit 10;
+
+-- 6. `orders` 테이블에서 오프셋 기반 페이지네이션된 목록을 조회합니다. 페이지 크기가 10이고 최신순일 때 5번째 페이지를 조회하세요.
+select * from orders 
+offset 40 limit 10;
+
+-- 7. `orders` 테이블에서 커서 페이지네이션된 목록을 조회합니다. 페이지 크기가 10이고 최신순일때, `id` 값을 기준으로 커서를 사용합시다. 커서의 값이 `42`일 때 다음 페이지를 조회하세요.
+select * from orders order by date desc ,time desc 
+offset 42 limit 10;
```

Comment:
이 친구는 커서를 잘못 쓴 것 같아요! id 기준으로 커서를 사용한다면 아래와 같이 작성되어야 하지 않을까 싶습니다!

```suggestion
-- 7. `orders` 테이블에서 커서 페이지네이션된 목록을 조회합니다. 페이지 크기가 10이고 최신순일때, `id` 값을 기준으로 커서를 사용합시다. 커서의 값이 `42`일 때 다음 페이지를 조회하세요.
SELECT * FROM orders
WHERE id > 42
ORDER BY id ASC, date DESC, time DESC
LIMIT 10;
```